packages/app/src/Navigator.tsx

```diff
@@ -82,6 +83,19 @@ function ProfileStack({
     );
 }
 
+const LiveTabsNavigator = createMaterialTopTabNavigator<LiveTabsParams>();
+
+export function LiveTabs(): ReactElement {
+    return (
+        <LiveTabsNavigator.Navigator initialRouteName="Comments">
+            <LiveTabsNavigator.Screen name="Comments" component={Comments} />
+            <LiveTabsNavigator.Screen name="Election">
+                {() => <></>}
+            </LiveTabsNavigator.Screen>
+        </LiveTabsNavigator.Navigator>
+    );
+}
+
 const MainTabNavigator = createMaterialTopTabNavigator<MainTabsParams>();
 export function MainTabsStack({
     mainnetProvider,
```

Comment:
NIT: Using the MainTabsParams['Live'] here isn't correct. This is my fault. Just did this to hack it through. Probably need to make a custom type for the main tabs.